x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -115,10 +117,15 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
     label: i18n.translate('xpack.ingestPipelines.processors.label.dateIndexName', {
       defaultMessage: 'Date index name',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.dateIndexName', {
-      defaultMessage:
-        'Uses a date or timestamp to add documents to the correct time-based index. Index names must use a date math pattern, such as my-index-yyyy-MM-dd.',
-    }),
+    description: {
+      defaultMessage: (
+        <FormattedMessage
+          id="xpack.ingestPipelines.processors.description.dateIndexName"
+          defaultMessage="Uses a date or timestamp to add documents to the correct time-based index. Index names must use a date math pattern, such as {value}."
+          values={{ value: <EuiCode inline>{'my-index-yyyy-MM-dd'}</EuiCode> }}
+        />
+      ),
+    },
```

Comment:
You will need to remove the `{ defaultMessage: ... }` wrapping the `<FormattedMessage ... />` component - you you will probably also get a TS error about this :)